Documentation/lists/lists.html

```diff
@@ -0,0 +1,160 @@
+  style="width: 475px; height: 355px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+        </div>
+
+        <div>
+            <h2>Stacked lists</h2>
+            <div class="list-container flex-center">
+            <ul class="stacked-list flex-col">
+                <li class="stacked-li dis-flex">Notification 1<i class="fa-solid fa-xmark"></i></li>
+                <li class="stacked-li dis-flex">Notification 2<i class="fa-solid fa-xmark"></i></li>
+                <li class="stacked-li dis-flex">Notification 3<i class="fa-solid fa-xmark"></i></li>
+                <li class="stacked-li dis-flex">Notification 4<i class="fa-solid fa-xmark"></i></li>
+                <li class="stacked-li dis-flex">Notification 5<i class="fa-solid fa-xmark"></i></li>
+            </ul>
+            </div>
+          </div>
+            <div>
+              <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253EStacked%2520lists%253C%252Fh2%253E%250A%253Cdiv%2520class%253D%2522list-container%2522%253E%250A%253Cul%2520class%253D%2522stacked-list%2520flex-col%2522%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522stacked-li%2520dis-flex%2522%253ENotification%25201%253Ci%2520class%253D%2522fa-solid%2520fa-xmark%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522stacked-li%2520dis-flex%2522%253ENotification%25202%253Ci%2520class%253D%2522fa-solid%2520fa-xmark%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522stacked-li%2520dis-flex%2522%253ENotification%25203%253Ci%2520class%253D%2522fa-solid%2520fa-xmark%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522stacked-li%2520dis-flex%2522%253ENotification%25204%253Ci%2520class%253D%2522fa-solid%2520fa-xmark%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522stacked-li%2520dis-flex%2522%253ENotification%25205%253Ci%2520class%253D%2522fa-solid%2520fa-xmark%2522%253E%253C%252Fi%253E%253C%252Fli%253E%250A%253C%252Ful%253E%250A%253C%252Fdiv%253E"
+  style="width: 875px; height: 355px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </div>
+          
+
+
+        </div>
+
+
+        </div>
+
+
+    <footer class="footer">
+        <div class="centered container">
+          <ul class="list-non-bullet">
+            <li class="list-inline">
+              <a
+                class="links"
+                target="_blank"
+                href="https://github.com/Priyanshi-Tiwari"
+                ><i class="fa-brands fa-github" style="font-size: 40px"></i
```

Comment:
avoid inline styles, instead increase the specificity and do it

Reply:
okay